tests/dummy/config/ember-try.js

```diff
@@ -8,18 +8,18 @@ module.exports = async function () {
     useYarn: true,
     scenarios: [
       {
-        name: 'ember-lts-3.28',
```

Comment:
we probs don't want to drop support for any of this -- but instead, expand support

Reply:
I agree but only because we want to release as a point release to fix the 5.0 release